tests/mock_callable_testslide.py

```diff
@@ -1111,6 +1111,10 @@ def mock_callable_dsl(self):
             def callable_target(self):
                 return getattr(self.real_target, self.callable_arg)
 
+            @context.memoize_before
+            def _original_target(self):
+                return getattr(self.real_target, self.callable_arg)
+
```

Comment:
Nit: I'd move this to line 607-608 so it is defined before it is referred for easier reading.